funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -169,6 +171,28 @@ def start_endpoint(*, name: str, endpoint_uuid: str | None):
     )
 
 
+@app.command(name="logout", help="Logout from all endpoints")
+@click.option("--environment", default=None, help="The environment to logout from")
+@common_options
+def logout_endpoint(environment: str | None):
+    """
+    Logout from all endpoints and remove cached authentication credentials
+
+    TODO Currently there does not seem to be credentials associated with
+     specific endpoints, so will just have to remove all tokens for
+     all environments using the default LoginManager.
+
+     If environment is specified, it will only clear credentials for
+     that environment (namespace in token_storage cached tokens)
+     Alternatively, environment can be retrieved from the endpoint
+     config.py if endpoint is specified, but that is a much more
+     convoluted way to get the environment
```

Comment:
I would be inclined not to include the environment param.  Long term we should be moving to environments that mirror (and connect) to Globus enviornments.  Ideally those can be controlled by the standard `GLOBUS_SDK_ENVIRONMENT` environment variable.  But for the moment we don't have that mapping in place.  This is also the only endpoint command that would expose that functionality (others need it set in the endpoint config).  We may at some point want to provide per-endpoint logout, or to support using the standard SDK environment variable, but for now say logout just logs you out of everything I think makes the most sense.

Reply:
Will modify logout message to be more clear